.uberenv_config.json

```diff
@@ -3,8 +3,8 @@
 "package_version": "develop",
 "package_final_phase": "initconfig",
 "package_source_dir": "../..",
-"spack_url": "https://github.com/spack/spack",
-"spack_commit": "23ddfba16ddc376c1a8ec77ffee731117b4fabf8",
+"spack_url": "https://github.com/davidbeckingsale/spack.git",
```

Comment:
Note @davidbeckingsale -- Looks like we're temporarily pointing at your fork of spack.
Please don't remove (rewrite) this commit until we can update back to spack proper.